content/reports/1000_IBM Power - Virtual Machines/010_All Power VMs.yaml

```diff
@@ -0,0 +1,87 @@
+---
+- MiqReport:
+    title: IBM PowerHMC VMs
+    rpt_group: Custom
+    rpt_type: Custom
+    priority:
+    db: ManageIQ::Providers::InfraManager::Vm
+    cols:
+    - name
+    - os_image_name
+    - ram_size
+    - entitled_processors
+    - processor_pin_policy
+    - num_cpu
+    - num_disks
+    - disk_size
+    - ipaddresses
+    - mac_addresses
+    include:
+      ext_management_system:
+        columns:
+        - name
+    col_order:
+    - ext_management_system.name
+    - name
+    - os_image_name
+    - ram_size
+    - entitled_processors
+    - processor_pin_policy
+    - num_cpu
+    - num_disks
+    - disk_size
+    - ipaddresses
+    - mac_addresses
+    headers:
+    - Provider Name
+    - Name
+    - OS Name
+    - RAM Size (MB)
+    - Entitled Processors
+    - Processor Pin Policy
+    - Number of CPUs
+    - Number of Disks
+    - Disk Size
+    - IP Addresses
+    - MAC Addresses
+    conditions:
+    order: Ascending
+    sortby:
+    group:
+    graph:
+    dims:
+    filename:
+    file_mtime:
+    categories: []
+    timeline:
+    template_type: report
+    where_clause:
+    db_options: {}
+    generate_cols:
+    generate_rows:
+    col_formats:
+    - :String Truncated to 50 Characters with Elipses (...)
+    - :String Truncated to 50 Characters with Elipses (...)
+    - :String Truncated to 50 Characters with Elipses (...)
+    - :Suffixed Megabytes (MB, GB)
```

Comment:
@jaywcarman Is this not the same thing causing https://github.com/ManageIQ/manageiq/issues/21823 ?

Reply:
@Fryguy this is the same, and I believe it would cause a problem if a chart were enabled. Without a chart, though, it works as expected.

Reply:
Oh I see...it's more the combination of a chart and the report 👍